src/components/modules/document/steps/DocumentTypes/DocumentTypes.view.vue

```diff
@@ -141,10 +141,10 @@ export default {
   margin-top: 4px;
 }
 .document-type-status.is-verified {
-  color: var(--endpass-ui-color-success);
+  color: #22a143;
```

Comment:
we should use colors from tags in vault (filter) and we have these colors in kit